src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task{
+    private static String SYMBOL = "D";
```

Comment:
Perhaps this should be a final variable as I presume the  SYMBOL is supposed to be a fixed constant